src/components/AssetItem/AssetItem.js

```diff
@@ -0,0 +1,178 @@
+
+    const {
+      isChange,
+      isHoveredEdit,
+      isHoveredTrash,
+      isDeleteDialog,
+    }: StateProps = this.state
+
+    return (
+      <JCard color='white' isBorderRadius isHover>
+        <div className='asset-item'>
+          <div className='info'>
+            {!isCustom ? (
+              <div className='symbol -icon'>
+                <JAssetSymbol symbol={symbol} color='gray' />
+              </div>
+            ) : (
+              <div className='symbol -text'>
+                <JText value={symbol} color='blue' weight='bold' size='normal' whiteSpace='wrap' />
+              </div>
+            )}
+            <div className='data'>
+              <div className='name'>
+                <JText value={name} color='dark' weight='bold' size='normal' whiteSpace='wrap' />
+              </div>
+              <div className='balance'>
+                <AssetBalance
+                  color='gray'
+                  size='small'
+                  symbol={symbol}
+                  balance={balance}
+                />
+              </div>
+            </div>
+          </div>
+          <div className='actions'>
+            {isCustom && (
+              <Fragment>
+                {!isDeleteDialog ? (
+                  <Fragment>
+                    <div
+                      className='overlay'
+                      onClick={handle(this.toggleChange)(!isChange)}
+                    />
+                    <div
+                      className={classNames('item', !isChange && '-hide')}
+                      onMouseEnter={handle(this.onHoverEdit)(true)}
+                      onMouseLeave={handle(this.onHoverEdit)(false)}
+                    >
+                      <JIcon
+                        size='medium'
+                        color={isHoveredEdit ? 'sky' : 'blue'}
+                        name='edit'
+                      />
+                    </div>
+                    <div
+                      className={classNames('item', !isChange && '-hide')}
+                      onMouseEnter={handle(this.onHoverTrash)(true)}
+                      onMouseLeave={handle(this.onHoverTrash)(false)}
+                      onClick={handle(this.toggleDeleteDialog)(!isDeleteDialog)}
+                    >
+                      <JIcon
+                        size='medium'
+                        color={isHoveredTrash ? 'sky' : 'blue'}
+                        name='trash'
+                      />
+                    </div>
+                    <div
+                      className={classNames('item -dotts', isChange && '-hide')}
+                      onClick={handle(this.toggleChange)(!isChange)}
+                    >
+                      <JIcon
+                        size='medium'
+                        color='gray'
+                        name='dots-full'
+                      />
+                    </div>
+                  </Fragment>
+                ) : (
+                  <Fragment>
```

Comment:
we really need to use [ButtonWithConfirm](https://github.com/jibrelnetwork/jwallet-web/tree/develop/src/components/ButtonWithConfirm) here